style.css

```diff
@@ -71,12 +84,10 @@ nav {
 
 .primary-text-container {
   height: 55px;
-  width: 327px;
 }
 
 .supporting-text-container {
   height: 87px;
-  width: 327px;
   margin-top: 12px;
   display: flex;
   align-items: center;
```

Comment:
- [ ] [Optional] Kindly fix the headline section as the name and the title is overlapping.
![Capture](https://user-images.githubusercontent.com/85095702/159092475-1298549c-5299-4ef1-a013-27d81c8d1a7f.PNG)